src/main/java/org/interledger/Fulfillment.java

```diff
@@ -7,104 +7,152 @@
 
 /**
  * The fulfillment of a {@link Condition}.
```

Comment:
I think it's helpful to describe what a `Fulfillment` and `Condition` actually are, per something like the above. Before, when we relied upon crypto-conditions project, people could look there and try to understand that stuff. Now that we don't have those links, it can be extra confusing.

I've noticed two impediments to outsider's understanding of these concepts. First, Hashlocks are a commonly understood concept if you're familiar with cryptocurrencies, but for those who aren't, the concept is foreign. Next, mapping that concept to the CryptoCondition spec is another leap.

So, simply talking about a "Condition" and and "Fulfillment" is often like speaking a foreign language, especially around people who are new to Interledger.

My $0.02.